GDJS/Runtime/pixi-renderers/spriteruntimeobject-pixi-renderer.js

```diff
@@ -131,3 +131,16 @@ gdjs.SpriteRuntimeObjectPixiRenderer.getAnimationFrameWidth = function(pixiTextu
 gdjs.SpriteRuntimeObjectPixiRenderer.getAnimationFrameHeight = function(pixiTexture) {
     return pixiTexture.height;
 };
+
+gdjs.SpriteRuntimeObjectPixiRenderer.setSprite = function(runtimeScene, imageName) {
+    this._spriteDirty = true;
+    this._sprite = new PIXI.Sprite(runtimeScene.getGame().getImageManager().getPIXITexture(imageName));
+};
+
+gdjs.SpriteRuntimeObjectPixiRenderer.prototype.setShader = function(pixiShader) {
+     this._sprite.shader = pixiShader;
+};
+
+gdjs.SpriteRuntimeObjectPixiRenderer.prototype.setFilter = function(pixiFilter) {
```

Comment:
This can be annoying in the future when adding multiple effects on objects. It isn't easy to remove an API as it might be used, and this api would be annoying/redundant/breaking the effects if we have multiple ones.

Reply:
We'll indeed clean this from this PR to be sure we don't add this API to set filters yet - it will be part of the work to enable filters on objects :)